docs/modules/ROOT/pages/erc721.adoc

```diff
@@ -66,24 +66,17 @@ trait IERC721ABI {
 :introspection: xref:introspection.adoc[Introspection]
 :eip165: https://eips.ethereum.org/EIPS/eip-165[EIP165]
 
-Although Starknet is not EVM compatible, this implementation aims to be as close as possible to the ERC721 standard by utilizing Cairo's short strings to simulate `name` and `symbol`.
+Although Starknet is not EVM compatible, this implementation aims to be as close as possible to the ERC721 standard.
 This implementation does, however, include a few notable differences such as:
 
-* `token_uri` returns a felt252 representation of the queried token's URI.
-The EIP721 standard, however, states that the return value should be of type string.
-If a token's URI is not set, the returned value is `0`.
-Note that URIs cannot exceed 31 characters at this time.
-See <<storing_erc721_uris,Storing ERC721 URIs>>.
 * ``interface_id``s are hardcoded and initialized by the constructor.
 The hardcoded values derive from Starknet's selector calculations.
 See the {introspection} docs.
 * `safe_transfer_from` can only be expressed as a single function in Cairo as opposed to the two functions declared in EIP721, because function overloading is currently not possible in Cairo.
 The difference between both functions consists of accepting `data` as an argument.
 `safe_transfer_from` by default accepts the `data` argument.
-If `data` is not used, simply pass an empty array.
-* `safe_transfer_from` is implemented such that the optional `data` argument mimics `bytes`.
-In Solidity, this means a dynamically-sized array.
-To be as close as possible to the standard, it accepts a dynamic array of felts.
+If `data` is not used, simply pass an empty `ByteArray`.
```

Comment:
A reminder that data probably makes more sense as a `Span<felt252>` because of Serde